src/builders.ts

```diff
@@ -219,6 +234,56 @@ function validateModifiers(isRequired: boolean, isList: boolean): void {
   }
 }
 
+function validateAndPrepareModelAttributes(
+  attributes?: string | string[]
+): string[] | null {
+  if (!attributes) {
+    return null;
+  }
+
+  if (typeof attributes === "string") {
+    attributes = attributes.split("@@");
+    attributes = attributes.map((attribute) =>
+      attribute.length ? `@@${attribute}`.trim() : ""
```

Comment:
```suggestion
      attribute.length ? `@@${attribute}`.trim() : null
```

Reply:
I used empty string to prevent error when concatenating null with strings